<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/notes.ico</normaloff>icon/notes.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="pe">
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_f">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_win"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_e">
    <property name="title">
     <string>편집(&amp;E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_del"/>
    <addaction name="separator"/>
    <addaction name="action_all_select"/>
   </widget>
   <widget class="QMenu" name="menu_o">
    <property name="title">
     <string>서식(&amp;O)</string>
    </property>
    <addaction name="action_font"/>
   </widget>
   <widget class="QMenu" name="menu_h">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_f"/>
   <addaction name="menu_e"/>
   <addaction name="menu_o"/>
   <addaction name="menu_h"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_new">
   <property name="text">
    <string>새로 만들기(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>열기(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>저장(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>다른 이름으로 저장(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>끝내기(&amp;X)</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>잘라내기(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>복사(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>붙여넣기(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_del">
   <property name="text">
    <string>삭제(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>글꼴(&amp;F)</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>메모장 정보(&amp;I)</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>실행 취소(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>다시 실행(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_all_select">
   <property name="text">
    <string>모두 선택(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_win">
   <property name="text">
    <string>새 창(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
